lib/rocketfuel_api/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module RocketfuelApi
-  VERSION = '0.1.1'
+  VERSION = '0.1.2'
```

Comment:
Updating the version in the branch is usually not a good idea. Especially if more than one branch is open at the same time. I usually update them in the master branch after merging.

Reply:
All right, good to know.